renovate.json

```
@@ -1,5 +1,10 @@
 {
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "github>cds-snc/renovate-config"
-  ]
+    "local>cds-snc/renovate-config"
```

my bad - copy/pasted you the wrong data:
```suggestion
    "github>cds-snc/renovate-config"
```